src/UI/Series/Details/InfoViewTemplate.hbs

```diff
@@ -55,6 +55,10 @@
             {{#if_eq seasonNumber compare="-1"}}
                 <span class="label label-default">{{title}}</span>
             {{/if_eq}}
+
+            {{#if_eq sceneSeasonNumber compare="-1"}}
```

Comment:
If both seasonNumber and sceneSeasonNumber = -1 then you'll see the title twice. Not sure if that can happen.